src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,7 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+import './UserInfo.scss';
+
+export const UserInfo = ({ user: { name, email } }) => (
+  <a className="UserInfo" href={`mailto: ${email}`}>
```

Comment:
```suggestion
  <a className="UserInfo" href={`mailto:${email}`}>
```
it should fix test
![Screenshot from 2023-09-11 12-57-45](https://github.com/mate-academy/react_static-list-of-posts-js/assets/33029864/83192cef-fad9-42c3-9f09-ee641fa017b4)